tools/depends/target/flatbuffers/Makefile

```diff
@@ -49,6 +49,9 @@ ifeq ($(PREFIX),)
 endif
 	rm -rf $(PLATFORM)/*; mkdir -p $(PLATFORM)
 	cd $(PLATFORM); $(ARCHIVE_TOOL) $(ARCHIVE_TOOL_FLAGS) $(TARBALLS_LOCATION)/$(ARCHIVE)
+ifeq ($(BUILD_FLATC),ON)
```

Comment:
no need to make this conditional

Reply:
no need to patch if we're not building flatc, but I can remove

Reply:
I know, but it doesn't break anything, so keep it simple.